test/data/testAppObj.json

```diff
@@ -581,32 +586,6 @@
                 "final_job_state": {
                   "authstrat": "kbaseworkspace",
                   "child_jobs": [],
-                  "condor_job_ads": {
```

Comment:
unnecessary for our purposes

Reply:
I think there's a bunch of that which may get returned from the EE2 server. We should watch for it when we pivot to the Python side of things.